src/main/java/model/Bet.java

```diff
@@ -0,0 +1,38 @@
+package model;
+
+import java.io.Serializable;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Bet implements Serializable {
```

Comment:
do we need Serializable here?

Reply:
The condition says to make a similar project as in the video. In the video class Bet implements Serializable, so I implemented it in class Bet and in class Person too.